packages/EasyApiToken/src/External/Auth0/TokenGenerator.php

```diff
@@ -76,6 +76,6 @@ public function generate(
             true
         ) : $this->secret;
 
-        return JWT::encode($payload, (string)$secret);
+        return JWT::encode($payload, (string)$secret, 'HS256');
```

Comment:
Please use the `EonX\EasyApiToken\Interfaces\AlgorithmsInterface::HS256` constant.